src/components/resource-section/resource-section/ResourceSection.js

```diff
@@ -1,17 +1,16 @@
-import ResourcePanel from '../resource-panel/ResourcePanel';
-import EvaluationList from '../evaluation-list/EvaluationList';
+import ResourceSidebar from '@components/resources-section/resource-sidebar/ResourceSidebar';
 import useGet from '@hooks/useGet';
 import { endpoints } from '@utils/endpoints';
 import { useRef } from 'react';
 
-const ResourceSection = ({ resourceId }) => {
+const ResourceSection = ({ visible, onHideHandler, resourceId }) => {
   const { data: resourceData, isLoading: isLoadingResource, isError: isErrorResource } = useGet(endpoints('resource', resourceId));
 
   const { data, isLoading: isLoadingEvaluation, isError: isErrorEvaluation } = useGet(endpoints('resourceEvaluation', resourceId));
 
   const { data: average_evaluation, isLoading: isLoadingAverage, isError: isErrorAverage, mutate: updateAverage } = useGet(endpoints('resourceAverage', resourceId));
 
-  const { data: evaluations, isLoading: isLoadingEvaluations, isError: isErrorEvaluations, mutate: updateEvaluations } = useGet(endpoints('resourceEvaluations', resourceId));
+  const { isLoading: isLoadingEvaluations, isError: isErrorEvaluations, mutate: updateEvaluations } = useGet(endpoints('resourceEvaluations', resourceId));
```

Comment:
Consider to preserve evaluations display as it is now, instead of erasing it.

Reply:
The evaluations display will be implemented on another PR to follow a sequential development.

Reply:
Ok